sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -582,6 +582,12 @@ object RapidsConf {
       .booleanConf
       .createWithDefault(true)
 
+  val CHUNKED_SUBPAGE_READER = conf("spark.rapids.sql.reader.chunked.subPage")
+      .doc("Enable a chunked reader for where possible for reading data that is smaller " +
+          "than the typical row group/page limit.")
```

Comment:
Can we explicitly state that "this config is respected by Parquet reader" similar to line 573 above? Similarly for `MAX_READER_BATCH_SIZE_BYTES` below. By doing so, we can know exactly what reader is using this config while the others are not.